src/main/drivers/bus_spi_ll.c

```diff
@@ -36,8 +36,6 @@
 #include "drivers/nvic.h"
 #include "drivers/rcc.h"
 
-spiDevice_t spiDevice[SPIDEV_COUNT];
```

Comment:
It's not a header file, and LL is not compiled with StdPeriph at the same time.
Are you sure this one should be removed rather than have internal linkage (`static`)?

Reply:
I see, `bus_spi_impl.h` has a declaration and `bus_spi.c` has a definition.